scripts/reprocess.py

```diff
@@ -19,6 +19,8 @@
         sys.exit("No tx_ids in file, exiting script")
 
     for tx_id in lines:
+        # Remove newline character at the end of the tx_id (if present)
+        tx_id = tx_id.rstrip()
```

Comment:
from reading the documentation, rstrip only gets rid of _whitespace_ by default. I would suggest that you might want to do `.rstrip('\r\n\t ')` instead.